File: c#/jogo.cs
using System;

class jogo
{
    static void Main()
    {
        Random random = new Random();

        int numero_escolhido = random.Next(1,101);
        int tentativas = 0;
        int n;

        while(tentativas < 7)
        {
            Console.WriteLine("Digite seu palpite (1 a 100): ");
            n = int.Parse(Console.ReadLine());
            tentativas++;
            if(n == numero_escolhido)
                Console.WriteLine($"\nParabéns acertaste em cheio em apenas {tentativas} tentativas\n");
            else   
                if(n < numero_escolhido)
                    Console.WriteLine("Baixo");
                else
                     Console.WriteLine("Alto");
            if(tentativas == 7 && n != numero_escolhido)
            {
                Console.WriteLine($"O número era: {numero_escolhido}. Fim de jogo!!");
            }
        }

    }
}

File: java/jogo.java
import java.util.Random;
import java.util.Scanner;

public class jogo {
    public static void main(String[] args) {
        Random rand = new Random();
        int numero_escolhido = rand.nextInt(100) + 1;
        Scanner scanner = new Scanner(System.in);
        int n = 0;
        int tentativas = 0;
        while (tentativas < 7) {
            
            System.out.print("Digite seu palpite (1 a 100): ");
            n = scanner.nextInt();
            tentativas ++;

            if (n == numero_escolhido) {
                System.out.println("\nParabéns acertaste em cheio em apenas "+ tentativas+" tentativas");
                break;
            } else {
                if(n < numero_escolhido){
                    System.out.println("Baixo");
                }
                    
                else{
                    System.out.println("Alto");
                }
                    
            }
        }
        if(tentativas == 7 && n != numero_escolhido)
            System.out.println("O número era:" + numero_escolhido + ". Fim de jogo!!");
        scanner.close();
    }
}